term/patterns/outlined-media-and-text.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Outlined Media and Text
+ * Slug: term/outlined-media-text
+ * Categories: media
+ * Block Types: core/media
+ */
+?>
+<!-- wp:columns {"style":{"border":{"width":"2px","radius":"8px"},"spacing":{"padding":{"top":"0","bottom":"0","left":"0","right":"0"},"margin":{"top":"0","bottom":"0"},"blockGap":{"top":"0","left":"0px"}}},"backgroundColor":"black"} -->
+<div class="wp-block-columns has-black-background-color has-background" style="border-width:2px;border-radius:8px;margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:column {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"blockGap":"0"},"color":{"background":"#edece8"},"border":{"width":"2px"}},"layout":{"type":"default"}} -->
+<div class="wp-block-column has-background" style="border-width:2px;background-color:#edece8;padding-top:0;padding-bottom:0"><!-- wp:paragraph {"style":{"spacing":{"margin":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|60","right":"var:preset|spacing|60"},"padding":{"top":"0","bottom":"0","left":"0","right":"0"}}}} -->
+<p style="margin-top:var(--wp--preset--spacing--60);margin-right:var(--wp--preset--spacing--60);margin-bottom:var(--wp--preset--spacing--60);margin-left:var(--wp--preset--spacing--60);padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">In his real life, Joel Gavins is actually a tax accountant. True story. You might think it’s the most boring job in the world, but he’ll never go out of business! Taxes are so terribly confusing that everyone needs someone else to do their taxes, that’s the whole point of the industry. So while your job might get taken over by AI, lobbying will always ensure that Joel has a nice cushy job and a working car.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:spacer {"height":"0px"} -->
+<div style="height:0px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"style":{"border":{"width":"2px"}}} -->
+<div class="wp-block-column" style="border-width:2px"><!-- wp:image {"id":21,"sizeSlug":"full","linkDestination":"none","className":"is-style-default"} -->
+<figure class="wp-block-image size-full is-style-default"><img src="http://block-theme-experiment.local/wp-content/uploads/2024/03/image-1.png" alt="" class="wp-image-21"/></figure>
```

Comment:
We would need to include the image file in the PR, and instead of linking like this, we should do it more like [here](https://github.com/WordPress/community-themes/blob/859fad0fc3c49ce8297736acc09db0b0e3ae5169/blue-note/patterns/blue-image-with-quote.php#L13), otherwise the image will only load on your environment :D

Reply:
I think both the image and the container of the paragraph block need a border radius to fit the design.